extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -26,6 +26,11 @@ export const msdb = 'msdb';
 export const MicrosoftDatatoolsSchemaSqlSql = 'Microsoft.Data.Tools.Schema.Sql.Sql';
 export const databaseSchemaProvider = 'DatabaseSchemaProvider';
 export const sqlProjectSdk = 'Microsoft.Build.Sql';
+export const problemMatcher = '$sqlproj-problem-matcher';
+export const sqlProjTaskType = 'sqlproj-build'
+export const dotnetBuild = 'dotnet build';
+export const runCodeAnalysisParam = '/p:RunSqlCodeAnalysis=true';
+export const runWithVerbose = 'v:detailed	';
```

Comment:
Are the extra spaces needed?

Reply:
seems accidentally did the tab, will remove it.